lib/wine_o_inventory_api_web/plugs/check_json_spec.ex

```diff
@@ -0,0 +1,26 @@
+defmodule WineOInventoryApiWeb.Plugs.CheckJsonSpec do
+    use WineOInventoryApiWeb, :controller
+    import Plug.Conn
+
+    def init(_params) do
+    end
+
+    def call(conn, _params) do
+        inspect_json(conn.method, conn)
+    end
+
+    def inspect_json("POST", conn) do
+        request_body = conn.body_params
+            if (request_body["data"] == nil || request_body["data"]["attributes"] == nil) do
+                conn
+                |> put_status(:bad_request)
+                |> json(%{errors: %{detail: "Bad Request"}})
+                |> halt()
+            end
```

Comment:
Why is this block indented so far to the right?

Reply:
Thank you for catching that! It was done by mistake.